src/net/sourceforge/kolmafia/session/QuestManager.java

```diff
@@ -2080,6 +2080,8 @@ else if (responseText.contains("Dozens of nearby smut orcs")) {
       QuestDatabase.setQuest(Quest.MAELSTROM, QuestDatabase.UNSTARTED);
 
       Preferences.setInteger("lastThingWithNoNameDefeated", KoLCharacter.getAscensions());
+    } else if (monsterName.equals("The SuperConductor")) {
```

Comment:
just `The Superconductor`, not `The SuperConductor`?

also in-game it starts with a space (` The Superconductor`), do we trim monster names already?

Reply:
We do.